app/components/moderation-base/component.js

```diff
@@ -1,30 +1,57 @@
 import { computed } from '@ember/object';
+import { inject as service } from '@ember/service';
 import Component from '@ember/component';
+
+import { task } from 'ember-concurrency';
 /**
  * The base for the provider moderation page (tabs, provider name, and breadcrumbs).
  *
  * Sample usage:
  * ```handlebars
- * {{moderation-base
- *   provider=provider
- *   pendingCount=pendingCount
- * }}
+ * {{moderation-base}}
  * ```
  * @class moderation-base
  * */
 export default Component.extend({
+    theme: service(),
+    store: service(),
+
     tabs: computed('pendingCount', function() {
         return [
             {
                 nameKey: 'global.moderation',
                 route: 'preprints.provider.moderation',
                 hasCount: true,
-                count: this.get('pendingCount') || 0,
+                count: this.get('pendingCount'),
             },
             {
                 nameKey: 'global.settings',
                 route: 'preprints.provider.settings',
             },
         ];
     }),
+
+    init() {
```

Comment:
This might make more sense as `didReceiveAttrs`. `init` makes tests harder and shouldn't be used for stuff that depends on incoming values.

Reply:
I moved it from a component and forgot to change it.